code/game/objects/items/cards_ids.dm

```diff
@@ -1104,6 +1145,9 @@ update_label("John Doe", "Clowny")
 		return .
 	var/obj/item/card/id/idcard = target
 	if(istype(idcard))
+		if(istype(idcard, /obj/item/card/id/paper))
+			to_chat(user, to_chat(user, "<span class='warning'>The pass beeps a warning about this not being a proper id."))
```

Comment:
How about something else (the pass has a speaker? and can talk)

How about: "The paper swipes along the pass. Nothing happens."

Also, missing `</span>`